### ISSUE TITLE
(IDEA NOT ISSUE)

### ISSUE THREAD

**Issue by PrimeKidMN** [QuiteAFancyEmerald/HolyUnblockerPublic#417]
Dear, whoever is reading this on the HolyUnblocker dev team,

I have an excellent idea for a holy unblocker. Nebula was my favorite unblocker of all time I would do advanced deployment with Freenom until it got removed and either way, it was a TON of work to get it running. Now all I have to do is run it through Koyeb and it works every time and is usually very fast. Unfortunately, it has not been kept well lately and I think it's dying. Holy Unblocker takes the stunning themes and advanced technology of Nebula and improves it, then it adds even more other stuff. I know I yapped a lot and y'all all probably finna crash out soon but hang in there. So basically my idea was you guys should add a login feature where you can save your account on holy unblocker with a passcode screen for added security. You would want to add saved like search history and other stuff but that would be advanced I think and I don't know if that's in y'all's wheelhouse. Right now Holy Unblocker feels cluttered and not like a web browser. Maybe when you click into the proxy it transforms into something that looks like Nebula and you can log into it. I know I did not word this well but it's like 4 am for me so I'm tired.

Thanks,
PrimeKidMN

**Comment by PrimeKidMN** [comment]
if anyone reads this at somepoint sry for writing in the issues section I just cant use discord

**Comment by QuiteAFancyEmerald** [comment]
Hey thanks for the detailed suggestion. Like really this is a lot. Holy Unblocker LTS predates Nebula so yeah design wise it can for sure use some improvement. I've considered having like a little widget like a proper browser just so it feels smoother as well as adding in more subtle animations. Fluidity has always been a core thing when it comes to Holy Unblocker.

As you can tell the site by nature right now is a mix of old and new just because of how old the actual source is hence why it technically isn't released yet (HU LTS). Almost five years now... its wild. I'll pin this issue so that as Yoct and I are designing it this isn't forgotten

**Comment by PrimeKidMN** [comment]
yeah no problem I was up at like 3am thinking about life and ya know ya know it turned into this lol

**Comment by PrimeKidMN** [comment]
rift riot at nebula gave me a hard time for giving them koyeb instructions since it only had local deploy instructions for nebula, which if you did not know it does now but the problem was I put it an issues which I get but I cant use discord so what am I supposed to do. (also yea sorry for putting an idea in issues but no one checks discussions so its probably fair.)

**Comment by PrimeKidMN** [comment]
I dont like nebula anymore. I researched for SO LONG as someone who was been taught node.js but completely forgot to get nebula to work on koyeb an then when I do nobody cares. who actually wants to PAY for a domain and servers from cloudflare and locally deploy it when you can just press a button. I cant teach my friends at school how to ssh into my servers which I pay for on cloudfare. wtf does that even mean like half the kids at school have like negative brain cells anyways and try to get them to learn how to setup a domain get around the schools dev mode policies and troubleshoot the issues that come up whn locally deploying. This is a crazy rant but like im just happy there is a proxy suitable for the people who want good quality sevice I guess you could say and a good product. You actually seem like a nice human being than the other ones. Also again im sorry for ranting I dont know anybody who actually knows about node.js and this is like the only positive interaction ive had on this platform. so again sorry sorry sorry for ranting

**Comment by rifting** [comment]
> rift riot at nebula gave me a hard time for giving them koyeb instructions since it only had local deploy instructions for nebula, which if you did not know it does now but the problem was I put it an issues which I get but I cant use discord so what am I supposed to do. (also yea sorry for putting an idea in issues but no one checks discussions so its probably fair.)

lol im going to add your instructions to the readme? how is adding what you asked a "hard time" lmaoo

**Comment by PrimeKidMN** [comment]
IM SORRY ):::: it was like 3 am when u wrote back I was in my emo depression phase cmon

**Comment by PrimeKidMN** [comment]
is that just me?

**Comment by PrimeKidMN** [comment]
Thats actually funny that you wrote on here  was not expecting that also yeah I guess the way you said it TO ME came off a little harsh but thinking about it that was just me also I wrote like so much on here I was not tryna be like KSI or nothing but im a lil sped when it comes to online discussions but yeah but again looking back that was all me so mb g

**Comment by PrimeKidMN** [comment]
oh yeah also by hard time I meant it in a way liek how I would bully my friend for like forgetting something in a funny way and not mean it but then forget it myself a minute later but like idk why even said that at all I need to go back and look what u said cuz if im being honest i forgot lol

**Comment by PrimeKidMN** [comment]
okay yea u just said "In the future" 😭 CUOT MY LIFE INTO PEUICES  I guess I was having a bad night lmao

**Comment by QuiteAFancyEmerald** [comment]
Mostly due to privacy concerns we probably won't adapt a search history feature; kinda defeats the purpose of the site lol. We have made considerable improvements however to making it feel more like a web browser such as adding in an omnibox; for clarity purposes for issues I'm going to close this. Feel free to open another issue if you have more ideas though
